readme.cchtml

```diff
@@ -0,0 +1,84 @@
+<h1>Runestone Interactive - Author Toolkit Tutorial:</h1>
+<p>This author toolkit is built using <strong>Github Codespaces</strong>, which
+    are development spaces that are hosted on the cloud. From a Codespace, you
+    can edit your files in a Visual Studio Code environment from within your
+    browser, and any processing power required is taken from the host machine
+    in the cloud. We are using this platform to set up a way for Runestone
+    authors to quickly and easily update their textbooks from anywhere.</p>
+<p>First, create a Github account if you do not already have one.</p>
+<p>Then, navigate to the textbook's repository (EX:
+    https://github.com/jbrock1999h/aata), click the green "Use this template"
+    at the top right, and "Open in Codespace" in the dropdown menu. <em>Note:
+        This step will eventually be replaced by an "edit me" button being
+        placed in the textbook that would directly link to the textbook's
+        Codespace.</em></p>
+<p><em><img src="images/Tutorial/Green_Button.png" alt="" width="849"
+            height="992"></em></p>
+<p>Github will take a short period of time (usually not more than 5 minutes) to
```

Comment:
We want this to be ideally < 30 seconds (all work done in the Docker build, just the boot time to get VSCode running.)